synthesis/codon/codon.go

```diff
@@ -338,6 +364,7 @@ func generateCodonTable(aminoAcids, starts string) codonTable {
 		triplet := string([]byte{base1[i], base2[i], base3[i]})
 		aminoAcidMap[aminoAcid] = append(aminoAcidMap[aminoAcid], Codon{triplet, 1})
 		if starts[i] == 77 { // M rune
+			stats.StartCodonCount[triplet]++
```

Comment:
This doesn't implement a check for the number of codons in a genome, just a count (which will be here, by definition, 1) of the codons that _can_ be start codons, not the weighted probability that any given one _will be_ a start codon, which is what we are looking for.